Cartfile

```diff
@@ -1,3 +1,3 @@
 github "Alamofire/Alamofire" ~> 4.0.0
-github "ReactiveCocoa/ReactiveSwift" "1.0.0-alpha.2"
-github "ReactiveX/RxSwift" "3.0.0-rc.1"
+github "ReactiveCocoa/ReactiveSwift" "1.0.0-alpha.3"
+github "ReactiveX/RxSwift" "3.0.0"
```

Comment:
It works as is now, but `""` tell carthage to use `git rev-parse`, which the dependency resolver often has issues with (à la https://github.com/Carthage/Carthage/issues/733)